api-lib/gql/__generated__/zeus/const.ts

```diff
@@ -2669,6 +2660,8 @@ export const AllTypesProps: Record<string, any> = {
     circle: 'circles_bool_exp',
     circle_id: 'Int_comparison_exp',
     created_at: 'timestamp_comparison_exp',
+    created_by: 'Int_comparison_exp',
```

Comment:
Actually now i'm noticing both are in here, and not totally understanding

Reply:
i also added a relationship `created_by_profile` to the profiles table using `created_by` as the column key.